src/Admin/Models/OrganizationSearchViewModel.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Bit.Admin.Models;
+
+public class OrganizationSearchViewModel : PagedModel<OrganizationSelectableViewModel>
```

Comment:
This name is likely to be confused with `OrganizationsModel` in the same folder. Not really sure how to distinguish them, though. You could make a new `Models/Provider` folder in this folder, or you could try making the name more specific, e.g. `UnassignedProviderOrganizationSearchViewModel` (suggestions on that one are welcome 😅 )